<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>verifyid</class>
 <widget class="QDialog" name="verifyid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_ppt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt; font-weight:500;</string>
   </property>
   <property name="text">
    <string>PASSPORT</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_license">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt; font-weight:500;</string>
   </property>
   <property name="text">
    <string>LICENSE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_C1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_l1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter license number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cc2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter registered date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 146, 28);
color: rgb(255, 255, 255);
border: none;
padding: 10,10,10,10;
margin: 10, 10, 10, 10;</string>
   </property>
   <property name="text">
    <string>SUBMIT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CC2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_p2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>510</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter expiry date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_L1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CC1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_P1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_c2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter registered date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_p1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter passport number</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_cc">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt; font-weight:500;</string>
   </property>
   <property name="text">
    <string>CITIZENSHIP CERTIFICATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_l2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter version number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_C2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_cer">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt; font-weight:500;</string>
   </property>
   <property name="text">
    <string>BIRTH CERTIFICATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_P2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_select">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>761</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#4e97d9;
font-size:30pt; font-weight:400;</string>
   </property>
   <property name="text">
    <string>Select the type of document you wish to provide, 
and enter folloing information.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cc1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter certificate number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_weNeedId">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#042d6d;
font-size:45pt; font-weight:600;</string>
   </property>
   <property name="text">
    <string>We need to verify your ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_c1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter certificate number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(4, 45, 109);
</string>
   </property>
   <property name="text">
    <string>In order to create an accout,
we need to be 100% sure that You are You.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_L2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
